bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/system/ITarget.java

```diff
@@ -28,7 +26,7 @@
  */
 public abstract class ITarget implements Cloneable, Serializable {
```

Comment:
The problem is that the target editor is based on ```SPTarget``` and ```ITarget```, also when dealing with the sequence model and getting parameters from the current environment this is all based on the old model. Since the target editor is my primary target here I think there's no way around doing this on the old model first.